Makefile

```diff
@@ -58,8 +58,10 @@ setup-pyenv:
 ensure-venv:
 	@./scripts/ensure-venv.sh
 
-ensure-pinned-pip: ensure-venv
-	$(PIP) install --no-cache-dir --upgrade "pip>=20.0.2"
+ensure-pinned-pip: ensure-venv upgrade-pip
+
+upgrade-pip:
+	$(PIP) install --no-cache-dir --upgrade "pip==20.0.2"
```

Comment:
This new target permits upgrading `pip` in the CI without ensuring that we're in a virtual environment (The `ensure-venv` target). This enables making the CI and `make install-py-dev` hit the same pip upgrade path (`install-py-dev -> ensure-pinned-pip -> upgrade-pip`).